### ISSUE TITLE
Add ARIA attributes to view data table link

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#9896]
The "View as data table" link in the screen reader information region should have an aria-expanded attribute that is updated, as well as a button role. We should also make sure focus is set to the table after pressing the button/link.
